eta/core/storage.py

```diff
@@ -613,6 +614,47 @@ def delete(self, file_or_folder_id):
         self._service.files().delete(
             fileId=file_or_folder_id, supportsTeamDrives=True).execute()
 
+    def delete_all_files_in_folder(self, folder_id):
+        '''Deletes all the files in the Google Drive folder and retains the
+        original folder.
+
+        Args:
+            folder_id: the ID of the Google Drive folder from which the files
+                are to be deleted
+
+        Returns:
+            an empty folder with no files
+        '''
+        files = self.list_files_in_folder(folder_id)
+        for f in files:
+            filename = f["name"]
+            try:
+                with etau.Timer() as t:
+                    self.delete(f["id"])
+                    logger.info(
+                        "File '%s' deleted from '%s' (%s)", filename, folder_id,
+                        t.elapsed_time_str)
+            except Exception as e:
+                if not skip_failures:
+                    raise GoogleDriveStorageClientError(e)
+
+        logger.info("All files deleted in %s", folder_id)
```

Comment:
Not necessarily. What if a deletion failed and you skipped it?

Reply:
yes, will add it. thanks.